### PR TITLE
Mv 2642

### PR DESCRIPTION
**Pull request by jmulford-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes

Similar to the OAS PR, making this a draft since I'm not sure when it should go live

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-MV-2642.s3-website-us-east-1.amazonaws.com/

**Comment by jmulford-bw** [comment]
No longer draft, will merge when given thumbs up from messaging team

File: js/custom.js [hunk 1]
```diff
@@ -62,7 +62,8 @@ module.exports = function ($) {
 	function hidePagesFromNav () {
 		var redirects = [
 			'International Overview',
-			'CSR Lookup API'
+			'CSR Lookup API',
+            'Get Messages'
```
Comment (jmulford-bw) [inline js/custom.js:66]: oh no. we have tabs in here 👎

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-MV-2642.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-MV-2642.s3-website-us-east-1.amazonaws.com/

**Pull request merged**